contrast_security/user_api/session_metadata_api.py

```diff
@@ -9,3 +9,27 @@ def __init__(self):
     def filter_app_session_metadata(self, org_uuid, app_id):
         path = '{org_uuid}/metadata/session/{app_id}/filters'.format(org_uuid=org_uuid, app_id=app_id)
         return self._post(path, json={})
+
+    def get_app_agentsessions(self, org_uuid, app_id):
+        path = '{orgUuid}/applications/{appId}/agent-sessions'.format(org_uuid=org_uuid, app_id=app_id)
+        return self._get(path, json={})
```

Comment:
Removed unused json={}. If get requires no input json empty json is not required.